preset/deep-learning/Dockerfile

```diff
@@ -429,11 +429,11 @@ ENV UCX_TLS=ud,sm,self
 
 USER dev
 
-RUN conda install -n pytorch-cpu -c intel/label/oneapi -c "${INTEL_CHANNEL}" -c conda-forge \
+RUN conda install -n pytorch-cpu -c https://software.repos.intel.com/python/conda -c "${INTEL_CHANNEL}" -c conda-forge \
```

Comment:
`intel/label/oneapi` can be removed. Hosted channel does not support labels at the moment. 
It should be
` conda create -yn pytorch-cpu -c "${INTEL_CHANNEL}" -c conda-forge \`